mcs/class/corlib/Test/System/EnvironmentTest.cs

```diff
@@ -231,6 +231,13 @@ public void SetEnvironmentVariable ()
 			Environment.SetEnvironmentVariable ("A3", "\0");
 			Assert.IsNull (Environment.GetEnvironmentVariables ()["A3"]);
 		}
+
+		[Test] // github issue #9839
+		public void MachineNameIsNotFullyQualifiedDomainName()
+		{
+			Assert.IsNotNull (Environment.MachineName);
+			Assert.AreEqual(Environment.MachineName.IndexOf("."), -1);
```

Comment:
Nit: missing spaces before parens here and in the method definition.

Nit: the expected parameter (-1) should be first.